logParser.js

```diff
@@ -45,30 +52,37 @@ function parseLog(text, itemList, logServer, client) {
         if (auctionWTS) {
+            // console.log("itemList", itemList);
+            // console.log("auctionWTS", auctionWTS);
             itemList.forEach(({item_name, user_id, user_name, price, server}) => {
+                // console.log('logServer = ', logServer, 'server = ', server, logServer === server);
                 if (server === logServer && auctionWTS.includes(item_name)) {
-                        // console.log('match found: ', item_name, user_id, user_name,  price, server);
-                        let filteredAuction = auctionWTS.slice(auctionWTS.indexOf(item_name), auctionWTS.length);
-                        // console.log("filtered auction = ", filteredAuction);
-                        let logPrice = parsePrice(filteredAuction, item_name.length);
-                        if (price === -1 && logPrice === null) {
-                            // console.log("match found - no price requirement", logPrice, price)
-                            let msg = {userId: user_id, userName: user_name, itemName: item_name, sellingPrice: logPrice, seller: auction_user, server: server, fullAuction: text};
-                            outgoing.push(msg);
-                        }
-                        else if (logPrice && logPrice <= price || price === -1) {
-                            // console.log("Meets price criteria", logPrice, price)
-                            let msg = {userId: user_id, userName: user_name, itemName: item_name, sellingPrice: logPrice, seller: auction_user, server: server, fullAuction: text};
-                            outgoing.push(msg);
-                        }
+                    // console.log('item name = ', item_name, 'price = ', price, 'server =', server, 'logServer = ', logServer);
+                    console.log('match found: ', item_name, user_id, user_name,  price, server);
+                    let filteredAuction = auctionWTS.slice(auctionWTS.indexOf(item_name), auctionWTS.length);
+                    // console.log("filtered auction = ", filteredAuction);
+                    let logPrice = parsePrice(filteredAuction, item_name.length);
+                    if (price === -1 && logPrice === null) {
+                        let msg = {userId: user_id, userName: user_name, itemName: item_name, sellingPrice: logPrice, seller: auction_user, server: server, fullAuction: text};
+                        console.log("match found - no price requirement", logPrice, price, msg)
+                        outgoing.push(msg);
+                    }
+                    else if (logPrice && logPrice <= price || price === -1) {
+                        let msg = {userId: user_id, userName: user_name, itemName: item_name, sellingPrice: logPrice, seller: auction_user, server: server, fullAuction: text};
+                        console.log("Meets price criteria", logPrice, price, msg)
+                        outgoing.push(msg);
                     }
                 }
-            ) 
+            }) 
```

Comment:
Wait what

Reply:
Wow this was confusing to look at a diff of.  The only change is that "})" is now on one line instead of two (because they are both opened on the same line in the forEach call above.